client/containers/App.js

```diff
@@ -5,15 +5,17 @@ import { Footer } from '../components/Footer';
 import { Home } from '../components/Home';
 import { Help } from '../components/Help';
 
+import problemsData from './problems_database'; //temp
+
 export class App extends Component {
     render() {
         return (
             <div>
                 <Header />
                 <Router history={browserHistory}>
                     <Route path="home" component={Home}/>
-                    <Route path="help" component={Help}/>
-                    <Redirect from="*" to="home" />
+                    <Route path="help" component={() => (<Help problems={problemsData}/>)}/>
```

Comment:
You definitely should use connect for this component to pass data from redux store into your Help component.
You should use examples from specification: 
https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options


In my opinion, for better decomposition you should create ProblemList component and wrap into react-redux connect component. And keep Home component simple and isolated from store.
For more information about architecture decomposition read: 
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.odxxhoo1t